test/relayer.js

```diff
@@ -1,19 +1,20 @@
 const ethers = require("ethers");
-const { AddressZero } = require("ethers").constants;
+const { formatBytes32String } = require("ethers").utils;
+const { parseRelayReceipt } = require("../utils/utilities.js");
 
 const Wallet = require("../build/BaseWallet");
-const TestModule = require("../build/TestModule");
-const TestModuleRelayerV2 = require("../build/TestModuleRelayerV2");
+const TestModule = require("../build/TestModuleRelayer");
 const Registry = require("../build/ModuleRegistry");
 const GuardianStorage = require("../build/GuardianStorage");
 const ApprovedTransfer = require("../build/ApprovedTransfer");
-const RecoveryManager = require("../build/RecoveryManager");
+const RecoveryManager = require("../build/RecoveryManager"); // non-owner only module
+const NftTransfer = require("../build/NftTransfer"); // owner only module
```

Comment:
We have `TestOnlyOwnerModule.sol` that could be used here instead

Reply:
I intentionally removed it to test behaviour with an actual OnlyOwnerModule we use. This could be a separate discussion altogether but I feel we overuse the TestContracts approach. Having a dedicated test contract is needed when it's a test case we can't achieve with the real-life contracts. Thus we keep test coverage reports up to date as well.